engine/src/flutter/impeller/entity/shaders/blending/porter_duff_blend.frag

```diff
@@ -20,13 +20,17 @@ layout(constant_id = 5) const float dst_coeff_src_color = 1.0;
 uniform f16sampler2D texture_sampler_dst;
 
 uniform FragInfo {
-  float16_t input_alpha;
-  float16_t output_alpha;
-  float tmx;
-  float tmy;
+  vec4 ai_ao_tmx_tmy;
+  vec4 source_rect;
+  float use_strict_source_rect;
 }
 frag_info;
 
+float input_alpha = frag_info.ai_ao_tmx_tmy.x;
+float output_alpha = frag_info.ai_ao_tmx_tmy.y;
+float tmx = frag_info.ai_ao_tmx_tmy.z;
+float tmy = frag_info.ai_ao_tmx_tmy.w;
```

Comment:
No abbreviations: https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Reply:
Oh I forgot, the glsl style guide says abbreviate everything.  Nevermind, change everything else then.

Reply:
(also yes, I will change this :) )

Reply:
Sorry, I was actually talking about the tmx, tmy variables.  The packed one was unfortunate but I figured the docstring would help.

```diff
- float tmx = frag_info.ai_ao_tmx_tmy.z;
- float tmy = frag_info.ai_ao_tmx_tmy.w;
+ vec2 tileMode = vec2(frag_info.ai_ao_tmx_tmy.z, frag_info.ai_ao_tmx_tmy.w);
```

Reply:
done